Orange/regression/random_forest.py

```diff
@@ -23,6 +25,31 @@ class RandomForestRegressionLearner(SklLearner, _FeatureScorerMixin):
     __wraps__ = skl_ensemble.RandomForestRegressor
     __returns__ = RandomForestRegressor
     name = 'random forest regression'
+    
+    """
+    n_estimators = The number of trees in the forest.
+    criterion = The function to measure the quality of a split.
+    max_depth = The maximum depth of the tree.
+    min_samples_split = The minimum number of samples required to split an internal node.
+    min_samples_leaf = The minimum number of samples in newly created leaves.
+    min_weight_fraction_leaf = The minimum weighted fraction of the input samples required to be at a leaf node.
+    max_features = The number of features to consider when looking for the best split:
```

Comment:
Please opt for proper [Napoleon-style](http://sphinxcontrib-napoleon.readthedocs.org/en/latest/index.html) docstrings in new code.